### PR TITLE
MGMT-10240 - Mask crio-wipe systemd unit

### PR DESCRIPTION
**Pull request by jparrill**
# Mask crio-wipe systemd unit

Masking this unit allows us to avoid the image cache deletion on the nodes once the enclosure is relocated.

The problem happens when a power cycle appears and the service crio-wipe deletes the node images stored in cache. 

This issue becomes a bigger issue, knowing that the cluster is disconnected and the nodes are using their own registry to boot the pods up.

This is the main reason for this improvement.

In addition I've refactored the save_files function using the @flaper87 method.

Signed-off-by: Juan Manuel Parrilla Madrid <jparrill@redhat.com>

Fixes [MGMT-10240](https://issues.redhat.com/browse/MGMT-10240)

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

- Needs Manual Testing on Flavio3
- Avoid CI Testing because of virtual worker deployment step broken (This step comes later)

### CODE REVIEW

File: finish-deployment/deploy.sh [hunk 1]
```diff
@@ -111,6 +113,14 @@ function store_rsa_secrets() {
...
 }
 
+function mask_systemd_units() {
+    # Function to mask dangerous systemd unit which could affect the relocation 
+    node=${1}
+    echo ">> Masking systemd units on node: ${node}"
+    echo ">>>> Unit: crio-wipe"
+    ${SSH_COMMAND} -i ${RSA_KEY_FILE} core@${node} "sudo systemctl mask crio-wipe"
+}
+
```
[the remark this answers is not in the document]
Comment (jparrill) [reply finish-deployment/deploy.sh]: Agree, moved into the BMH ignitionConfigOverride

**Review by flaper87 — APPROVED** [review]
/lgtm

**Review by flaper87 — APPROVED** [review]
/lgtm

**Comment by jparrill** [comment]
- Tests passed
- Worker still broken in virtual
- Last task executed manually:
image

**Pull request merged**